backend-api/src/functions/services/events/eventService.ts

```diff
@@ -32,7 +32,8 @@ export class EventService implements IEventService {
           MessageBody: JSON.stringify(txmaEvent),
         }),
       );
-    } catch {
+    } catch (error) {
+      console.log(error);
```

Comment:
This _feels_ like something that is reproducible from running the tests from your machine  👀